PhotoKingdom/app/src/main/java/ca/senecacollege/prj666/photokingdom/MapContainerFragment.java

```diff
@@ -239,6 +248,9 @@ public void getNearbyAttractions(double radiusMeters, double lat, double lng){
             Log.e(TAG, e.getMessage());
         }
 
+        // Clear old markers and shapes
+        mGoogleMap.clear();
```

Comment:
In my new PR, I have also added the mGoogleMap.clear() inside the OnCameraIdle function because I only want the markers to clear if it's at a close zoom level, and I don't want it to clear if we're at City/Continent zoom level, because of the way I have to add markers for the City/Province/Country/Continent owns, the markers need to stay. But I still have to fix the code up a bit more to make sure it clears the first time before going to CITY_ZOOM level or CONTINENT_ZOOM level. but anyway, just a comment, we can leave urs, and I will need to change it soon.